tools/src/test/java/org/apache/kafka/trogdor/workload/PayloadGeneratorTest.java

```diff
@@ -24,10 +24,12 @@
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
 
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
-
+import static org.junit.Assert.assertNull;
 
 public class PayloadGeneratorTest {
```

Comment:
should we have a test of the constructor failing when someone tries to set up a configuration that doesn't sum to 100 ?